web/src/beta/components/ListItem/index.tsx

```diff
@@ -71,17 +71,17 @@ const Inner = styled.button<{ border?: boolean; isSelected?: boolean; clamp?: Cl
   width: 100%;
   min-height: 38px;
   align-items: center;
-  border: 1px solid ${({ border }) => (border ? "#383838" : "transparent")};
+  border: 1px solid ${({ border, theme }) => (border ? theme.outline.weakest : "transparent")};
   border-radius: ${({ clamp }) =>
     clamp === "left" ? "0 6px 6px 0" : clamp === "right" ? "6px 0 0 6px" : "6px"};
   box-sizing: border-box;
   padding: 8px 20px 8px 4px;
   background: ${({ theme, isSelected }) => (isSelected ? theme.select.main : "inherit")};
   transition: all 0.3s;
 
-  ${({ isSelected }) => isSelected && `background-color: #3B3CD0;`}
+  ${({ isSelected, theme }) => isSelected && `background-color:` + theme.select.main}
```

Comment:
@nina992 It seems this component has both `background` and `background-color`. Please remove here since it isn't needed